File: app/src/main/java/com/vsa/filmoteca/view/DetailView.java
package com.vsa.filmoteca.view;

/**
 * Created by seldon on 13/03/15.
 */
public interface DetailView {

    void hideContent();

    void showContent();

    void showProgressDialog();

    void hideProgressDialog();

    void showMovieTitle(String title);

    void updateWidget();

    void stopRefreshing();

    void showTimeOutDialog();

    void launchBrowser(String url);

    void showShareDialog();

    void showAboutUs();

    void setWebViewContent(String html, String baseUrl);

    void navitgateToComments(String title);

    void onBackPressed();

}

File: app/src/main/java/com/vsa/filmoteca/presentation/view/DetailView.kt
package com.vsa.filmoteca.presentation.view

import com.vsa.filmoteca.core.presentation.view.LoadingView

/**
 * Created by seldon on 13/03/15.
 */
interface DetailView : LoadingView {

    fun hideContent()

    fun showContent()

    fun showMovieTitle(title: String)

    fun updateWidget()

    fun stopRefreshing()

    fun showTimeOutDialog()

    fun launchBrowser(url: String)

    fun showErrorNoInternet()

    fun showShareDialog()

    fun showAboutUs()

    fun setWebViewContent(html: String, baseUrl: String)

}
